iina/PrefKeyBindingViewController.swift

```diff
@@ -155,101 +144,74 @@ class PrefKeyBindingViewController: NSViewController, PreferenceWindowEmbeddable
     mappingController.remove(sender)
     NotificationCenter.default.post(Notification(name: .iinaKeyBindingChanged))
   }
+  
+  // Check whether or not a new config file with provided filename should be created.
+  // Returns the path of the new config if could be created; nil otherwise
+  private func checkNewConfigFile(with filename: String) -> String? {
```

Comment:
It's returning a value, so better called `newConfigFilePath(forName filename:)` or something.

Also, better to use Swift doc comments.